scss/components/_polyline.scss

```diff
@@ -0,0 +1,33 @@
+.blog {
```

Comment:
It would better to use separately element for polyline and don't get attached to some special elements like `portfolio` or `blog `